examples/simple.rs

```diff
@@ -65,25 +65,37 @@ fn start_example_client_code() -> Result<()> {
     info!("[example] Send register output request.");
     common::post(&json!({
         "type": "register",
-        "entity_type": "output_stream",
+        "entity_type": "output",
         "output_id": "output_1",
         "port": 8002,
         "ip": "127.0.0.1",
-        "resolution": {
-            "width": VIDEO_RESOLUTION.width,
-            "height": VIDEO_RESOLUTION.height,
+        "video": {
+            "resolution": {
+                "width": VIDEO_RESOLUTION.width,
+                "height": VIDEO_RESOLUTION.height,
+            },
+            "encoder_preset": "medium",
         },
-        "encoder_settings": {
-            "preset": "medium"
+        "audio": {
+            "sample_rate": 44_100,
+            "rtp_clock_rate": 48_000,
         }
     }))?;
 
-    info!("[example] Send register input request.");
+    info!("[example] Send register input video request.");
     common::post(&json!({
         "type": "register",
-        "entity_type": "input_stream",
+        "entity_type": "input",
         "input_id": "input_1",
-        "port": 8004
+        "port": 8004,
+        "video": {
+            "codec": "h264",
+        },
+        "audio": {
+            "codec": "aac",
+            "sample_rate": 44_100,
+            "rtp_clock_rate": 48_000
```

Comment:
cant find the relevant fragment of an rfc, but is having two values here necessary?

Reply:
`sample_rate` describes audio params
`rtp_clock_rate` describes is used for RTP timestamps

As I understand, I need `rtp_clock_rate` for correct RTP packet depayloading and sample rate for decoding actual audio (rtp packet payload).